great_expectations/render/renderer/inline_renderer.py

```diff
@@ -0,0 +1,127 @@
+            expectation_type = render_object.expectation_config.expectation_type
+            renderer_prefix = "atomic.diagnostic"
+        else:
+            raise ValueError(
+                f"object must be of type ExpectationConfiguration or ExpectationValidationResult, but an object of type {type(render_object)} was passed"
+            )
+
+        renderer_names: List[str] = get_renderer_names_with_renderer_prefix(
+            object_name=expectation_type,
+            renderer_prefix=renderer_prefix,
+        )
+
+        rendered_content: List[
+            RenderedAtomicContent
+        ] = self._get_atomic_rendered_content_from_renderer_names(
+            render_object=render_object,
+            renderer_names=renderer_names,
+            expectation_type=expectation_type,
+        )
+
+        return rendered_content
+
+    def _get_atomic_rendered_content_from_renderer_names(
+        self,
+        render_object: Union[ExpectationConfiguration, ExpectationValidationResult],
+        renderer_names: List[str],
+        expectation_type: str,
+    ) -> List[RenderedAtomicContent]:
+        default_prescriptive_renderer_name: str = "atomic.prescriptive.kwargs"
+        rendered_content: List[RenderedAtomicContent] = []
+
+        for renderer_name in renderer_names:
+            if renderer_name != default_prescriptive_renderer_name:
+                try:
+                    renderer_rendered_content = (
+                        self._get_renderer_atomic_rendered_content(
+                            render_object=render_object,
+                            renderer_name=renderer_name,
+                            expectation_type=expectation_type,
+                        )
+                    )
+                    rendered_content.append(renderer_rendered_content)
+                except Exception:
+                    pass
+
+        if len(rendered_content) == 0 and isinstance(
+            render_object, ExpectationConfiguration
+        ):
+            renderer_rendered_content = (
+                InlineRenderer._get_renderer_atomic_rendered_content(
+                    render_object=render_object,
+                    renderer_name=default_prescriptive_renderer_name,
+                    expectation_type=expectation_type,
+                )
+            )
+            rendered_content.append(renderer_rendered_content)
+
+        return rendered_content
+
+    @staticmethod
+    def _get_renderer_atomic_rendered_content(
+        render_object: Union[ExpectationConfiguration, ExpectationValidationResult],
+        renderer_name: str,
+        expectation_type: str,
+    ) -> Optional[RenderedAtomicContent]:
+        renderer_rendered_content: Optional[RenderedAtomicContent] = None
+        renderer_tuple: Optional[tuple] = get_renderer_impl(
+            object_name=expectation_type, renderer_type=renderer_name
+        )
+        if renderer_tuple is not None:
```

Comment:
Is it possible to make this a defined type (like a `namedtuple`)?

Reply:
Yes, good idea!

Reply:
Per our offline conversation, not going to do this as it is a breaking change for cloud to return something different from `get_renderer_impl`.